rest/route/user_test.go

```diff
@@ -775,6 +775,7 @@ func TestRenameUser(t *testing.T) {
 			assert.NotEqual(t, newUsr.APIKey, newUsrFromDb.GetAPIKey())
 			assert.Equal(t, "new_me@still_awesome.com", newUsrFromDb.Email())
 			assert.Equal(t, newUsrFromDb.PatchNumber, 8)
+			assert.Equal(t, 12, newUsrFromDb.Settings.GithubUser.UID)
```

Comment:
Re the unique index thing I asked about below, wouldn't the `UserAlreadyExists` test pass regardless of whether we made this change because there's no unique index on GitHub UID for this test? This test would pass before this change was made, and continues to pass after this change. If we wanted to double-check that the uniqueness constraint works with the code, we would want to build a unique index in this test on the GitHub UID to simulate the condition that no two users can hold the same GitHub UID at the same time as proof that it's testing under the right conditions.